courses/models_test.py

```diff
@@ -268,7 +277,7 @@ def test_course_available_runs():
     """enrolled runs for a user should not be in the list of available runs"""
     user = UserFactory.create()
     course = CourseFactory.create()
-    runs = CourseRunFactory.create_batch(2, course=course)
+    runs = CourseRunFactory.create_batch(2, course=course, live=True)
```

Comment:
all these tests are where `live=True`, can we add a test with `live=False`, which should not be returned.

Reply:
Sure